.github/workflows/build.yml

```diff
@@ -1,26 +1,325 @@
+          # going out. The stable channel implies that all tests and scans have
+          # completed successfully, so these images should be _stable_ but are not
+          # intended for production use.
+          #
+          # Here we have two example dev tags. The first (ending -dev) is a tag
+          # that will be updated over-and-over as new builds arrive in stable.
+          #
+          # The second (using the git SHA) will produce a new separate tag for
+          # each commit that is built. (These can still be overridden if the same
+          # commit is built successfully a second time, but that is a less likely
+          # scenario.) These kinds of dev tags are useful if you want to be able
+          # to use Docker images built from those specific commits.
+          #
+          # NOTE: dev_tags MUST publish to the 'hashicorppreview' DockerHub org, it
+          # will fail to any other DockerHub org or registry. You can optionally
+          # prepend docker.io
+          dev_tags: |
+            hashicorppreview/${{ env.repo }}:${{ env.version }}-dev
+            docker.io/hashicorppreview/${{ env.repo }}:${{ env.version }}-${{ github.sha }}
+
+  build-docker-alternate:
+    name: Docker ${{ matrix.arch }} alternate release build
+    needs:
+      - get-product-version
+      - build-linux
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        arch: ["arm", "arm64", "386", "amd64"]
+    env:
+      repo: ${{ github.event.repository.name }}
+      version: ${{ needs.get-product-version.outputs.product-version }}
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: hashicorp/actions-docker-build@v1
+        with:
+          version: ${{ env.version }}
+          target: release-alternate
+          arch: ${{ matrix.arch }}
+          tags: |
+            docker.io/hashicorp/${{ env.repo }}-alternate:${{ env.version }}
+            986891699432.dkr.ecr.us-east-1.amazonaws.com/hashicorp/${{ env.repo }}-alternate:${{ env.version }}
+          dev_tags: |
+            hashicorppreview/${{ env.repo }}-alternate:${{ env.version }}-dev
+            docker.io/hashicorppreview/${{ env.repo }}-alternate:${{ env.version }}-${{ github.sha }}
+
+  # build-docker-redhat is an example of how to build a Red Hat Certified Container (RHCC) Image.
+  # This means it uses the 'redhat_tag' input, instead of 'tags' and 'dev_tags'.
+  #
+  # It's important to note that there are other important differences with RHCC images, compared
+  # with calls using the 'tags' and optionally 'dev_tags' inputs:
+  #
+  #   - They can only be built for a single architecture,
+  #     so the 'arch' input is hard-coded to 'amd64' in this case.
+  #   - Therefore, we don't use a matrix to expand over architectures,
+  #     when using the 'redhat_tag' input.
+  #   - We can only specify a single 'redhat_tag' not multiple tags,
+  #     as with the 'tags' and 'dev_tags' inputs.
+  build-docker-redhat:
```

Comment:
You will only need this if you publish [ubi images](https://hashicorp.atlassian.net/wiki/spaces/ENSV/pages/2324269307/March+31st+2022+-+Red+Hat+Certified+Container+Images) for consul-template (which I don't think you do currently).

Reply:
Nope. Nuked.